packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -145,8 +148,16 @@ class FlutterSimpleTemplate extends Template {
     files['flutter.yaml'] = _flutterYaml;
     files['pubspec.yaml'] = _pubspec;
     files['README.md'] = _readme;
-    files['android/AndroidManifest.xml'] = _apkManifest;
     files['lib/main.dart'] = _libMain;
+
+    // Android files.
+    files['android/AndroidManifest.xml'] = _apkManifest;
+    // Create a file here, so we create the directory for the user (and it gets
+    // committed with git).
+    files['android/res/readme.md'] = _androidResReadme;
```

Comment:
Hmm, sort of. I can change this to match the root readme name. I've just never been a fan of all caps file names.